examples/bzlmod/cross_compile/BUILD.bazel

```diff
@@ -0,0 +1,33 @@
+load("@rules_rust//rust:defs.bzl", "rust_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+rust_binary(
+    name = "hello_world_aarch64",
```

Comment:
Can you throw in a `sh_test` or `rust_test` which takes this as a data dependency, runs `file` on it, and asserts on the output to show that it was _actually_ compiled for the correct target platform?

Reply:
Done!